tools/sdk-testgen/packages/autorest.testmodeler/src/core/model.ts

```diff
@@ -599,24 +594,26 @@ export class TestCodeModeler {
         }
     }
 
+    public createApiScenarioLoaderOption(fileRoot: string) {
+        const options = {
+            useJsonParser: false,
+            checkUnderFileRoot: false,
+            fileRoot: fileRoot,
+            swaggerFilePaths: this.testConfig.getValue(Config.inputFile),
+            eraseXmsExamples: false,
+        };
+        return { ...options, ...this.testConfig.getValue(Config.apiScenarioLoaderOption, {}) };
+    }
+
     public async loadTestResourcesFromConfig(session: Session<TestCodeModel>, fileRoot: string) {
         for (const testResource of this.testConfig.getValue(Config.testResources)) {
-            if (fs.existsSync(path.join(fileRoot, testResource[Config.test]))) {
```

Comment:
why do you remove this check?

Reply:
The old oav only support scenarios in local file system, so this can help filter out wrong configuration.

The new oav support remote swagger&scenario, so I remove this check since don't find an effect way to check exists for remote files. And we have code to catch error thrown by the scenario loader when there is wrong configuration, so I think removing this won't break anything.